src/components/github-htmlview.component.js

```diff
@@ -388,13 +389,13 @@ export class GithubHtmlView extends Component {
       };
 
       if (_node.type === 'text') {
-        if (_node.data === '\n') {
+        if (_node.data.trim() === '' && _node.data.includes('\n')) {
```

Comment:
@machour Is it possible that `_node.data` contains more than one `\n`? Then we need to render as multiply empty lines.

If not, everything is handled very well.

Reply:
Markdown only takes a single line break into account, so it should be okay